packages/notification-service/config/config.mainnet.env

```diff
@@ -1,10 +0,0 @@
-ENVIRONMENT=mainnet
-PORT=80
-FIREBASE_PROJECT_ID=celo-mobile-mainnet
-BLOCKSCOUT_API=https://rc1-blockscout.celo-testnet.org/api?
-NODE_TLS_REJECT_UNAUTHORIZED=0
-DEFAULT_LOCALE=en
-NOTIFICATION_TTL_MS=604800000
-POLLING_INTERVAL=1000
-EXCHANGE_POLLING_INTERVAL=1800000
-WEB3_PROVIDER_URL=https://rc1-forno.celo-testnet.org
```

Comment:
I know this was deleted but just wanted to double check that the new `.env` file is pointing to the new forno endpoint: `forno.celo.org` is it I think

Reply:
Good. catch, I updated it in the indexer as well